### PR TITLE
Simplify an algorithm for getting width and height

### LINKED ISSUE #77: CatBlock is not replacing ads with cats
A regression of #66 has been identified, leading to not displaying replaced images of cats.

### PR DESCRIPTION
**Pull request by tomasko126**
Fixes #77 

@kpeckett This is an important fix. Can you please visit neowin.net in order to check, whether you see images of cats or not? Thanks.

### CODE REVIEW

**Comment by itskdog** [comment]
No ads, but no cats either.

screenshot 2016-09-10 at 10 14 26 am

File: catblock/contentscript.js [hunk 1]
```diff
@@ -116,14 +116,10 @@ var picinjection = {
...
-            return intFor(el.getAttribute(prop));
+        if (typeof el.getAttribute !== "function") {
+            return intFor(window.getComputedStyle(el)[prop]);
         } else {
-            try {
-                return intFor(window.getComputedStyle(el)[prop]);
-            } catch(e) {
-                return undefined;
-            }
+            return intFor(el.getAttribute(prop) || window.getComputedStyle(el)[prop]);
```
Comment (itskdog) [inline catblock/contentscript.js:122]: If we've already worked out that `el.getAttribute` is not a function, why are we still trying it?

Comment (tomasko126) [reply catblock/contentscript.js:122]: @kpeckett This block of the code will run when `getAttribute` is a function.

**Comment by itskdog** [comment]
Neowin looks good, but Reddit has way too many cats at the top. If this is still a bug, then continue fixing, otherwise go ahead and merge it.

**Comment by tomasko126** [comment]
@kpeckett The issue with too many cats is being tracked in #79.

**Pull request merged**